analyses/cell-type-wilms-tumor-14/.gitignore

```
@@ -5,3 +5,6 @@
 # Ignore the scratch directory (but keep it present)
 /scratch/*
 !/scratch/.gitkeep
+
+# Plots for certain steps should not be committed
+/plots/01_anchor_transfer_seurat/*
```

Only exclude pdfs to allow other plots to be included
```suggestion
/plots/01_anchor_transfer_seurat/**.pdf
```